config/config.go

```diff
@@ -847,6 +848,60 @@ func setupTranslation() error {
 	})
 }
 
+// If the config file defines a "ConfigLocations" key and a list of corresponding directories, we parse all the yaml
+// files in those directories according to directory-scoped lexicographical order. This allows users/admins to split
+// their configuration across multiple directories/files.
+func handleContinuedCfg() error {
+	if cfgDirs := viper.GetStringSlice("ConfigLocations"); len(cfgDirs) != 0 {
```

Comment:
For better readability, can we pull the body of this if statement out by simply returning the function if the length of the slice is 0?